packages/pilot/src/containers/Refund/index.js

```diff
@@ -67,10 +69,38 @@ const getBoletoIdentificationData = applySpec({
   refundAmount: getDataAmount,
 })
 
+const buildRecipientsOptions = (t, recipients) => {
+  const options = [
+    {
+      name: t('pages.refund.keep_split_rules'),
+      value: 'same',
+    },
+    ...recipients.map(({ id, name }) => ({
+      name,
+      value: id,
+    })),
+  ]
+
+  const isSplitting = recipients.length > 1
+
+  return {
+    isSplitting,
+    options,
+  }
+}
```

Comment:
apenas uma sugestão para facilitar a leitura de como é feita as opções

```suggestion
const recipientList = recipients.map(({ id, name }) => ({
      name,
      value: id,
    }))
const buildRecipientsOptions = (t, recipients) => {
  const options = [
    {
      name: t('pages.refund.keep_split_rules'),
      value: 'same',
    },
    ...recipientList,
  ]

  const isSplitting = recipients.length > 1

  return {
    isSplitting,
    options,
  }
}
```